src/Covid19Dashboard/PlotChart/index.jsx

```diff
@@ -40,62 +60,104 @@ function formatChartData(plots) {
+  handleMouseEnter = (o) => {
+    const { dataKey } = o;
+    const { width } = this.state;
+
+    this.setState({
+      width: { ...width, [dataKey]: 2 },
+    });
+  }
+
+  handleMouseLeave = (o) => {
+    const { dataKey } = o;
+    const { width } = this.state;
+
+    this.setState({
+      width: { ...width, [dataKey]: 1 },
+    });
+  }
+
   renderTooltip = (props) => {
     const monthNames = ['Jan', 'Feb', 'Mar', 'April', 'May', 'Jun', 'Jul', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
     const date = new Date(props.label);
     return (
       <div className='map-chart__tooltip'>
-        <p>{monthNames[date.getMonth()]} {date.getDate()}, {date.getFullYear()}</p>
+        <p>{monthNames[date.getUTCMonth()]} {date.getUTCDate()}, {date.getUTCFullYear()}</p>
         {
           props.payload.map((data, i) => (
-            <p style={{ color: data.stroke }} key={i}>{data.name}: {data.value.toExponential()}</p>
+            <p style={{ color: data.stroke }} key={i}>{data.name}: {data.value}</p>
           ))
         }
+        <div style={{ marginTop: `${2}em` }}><p>{this.props.description}</p></div>
       </div>
     );
   }
 
   render() {
     const chartData = formatChartData(this.props.plots);
+    const { width } = this.state;
 
     return (
       <div className='plot-chart'>
         <p className='plot-chart__title'>
           {this.props.title}
         </p>
-        <ResponsiveContainer>
+        <ResponsiveContainer height={250}>
           <LineChart
             data={chartData.data}
             margin={{
               top: 5, right: 30, left: 20, bottom: 5,
             }}
           >
-            <CartesianGrid strokeDasharray='3 3' />
+            <CartesianGrid
+              horizontal={true}
+              vertical={false}
+              strokeDasharray='3 3'
+            />
             <XAxis
               dataKey='date'
               // label={this.props.xTitle}
               tick={<CustomizedAxisTick />}
-              interval={1}
+              ticks={chartData.ticks}
+              // interval='preserveStartEnd'
+              interval={0}
             />
             <YAxis
               label={{
                 className: 'plot-chart__y-title',
                 value: this.props.yTitle,
                 angle: -90,
                 position: 'insideLeft',
+                offset: -10
               }}
               type='number'
               domain={[0, 'auto']}
             />
             <Tooltip content={this.renderTooltip} />
-            <Legend />
-            <Line type='monotone' dataKey={this.props.plots[0].name} stroke='#8884d8' dot={false} />
-            <Line type='monotone' dataKey={this.props.plots[1].name} stroke='#aa5e79' dot={false} />
+            <Legend onMouseEnter={this.handleMouseEnter} onMouseLeave={this.handleMouseLeave} />
+            {
+              this.props.plots.map((entry, index) => {
+                const color = COLORS[index];
```

Comment:
is it possible that we have more entries than COLORS.length? if so, better put some guarding conditions here or use `% COLORS.length`